velox/docs/monthly-updates/april-2023.rst

```diff
@@ -0,0 +1,75 @@
+********************
+April 2023 Update
+********************
+
+
+Documentation
+=============
+
+ * Add :doc:`documentation on UnsafeRow serialization <../develop/unsaferow>`.
+
+
+Core Library
+============
+
+ * Add support for peeling past a loaded lazy layer.
+ * Add support for companion function generation for aggregations.
+ * Add support to suppress errors for unused arguments of null-propagating functions.
+
+
+Presto Functions
+================
+
+ * Add :func:`timezone_minute`,   :func:`timezone_hour` functions.
+ * Add :func:`date`,   :func:`random()`,  :func:`sequence`,  :func:`from_base64url` functions.
+ * Add support for DECIMAL comparison functions.
```

Comment:
This should probably be "Implement DECIMAL comparisons as vector functions".
Decimal comparisons were already implemented as simple functions. The recent change is that they have been re-implemented as vector functions.